contracts/managers/AssetPairManagerV2.sol

```diff
@@ -316,7 +330,7 @@ contract AssetPairManagerV2 is
     {
         // If RebalancingSetToken in valid state and new allocation different from last known allocation
         // then return true, else false
-        return rebalancingSetTokenInValidState()
+        return rebalancingAllowed()
```

Comment:
Think this name is too general since this is just checking that it won't fail on the RebalancingSetToken, while there other checks to make sure it won't fail on the on the Manager (i.e. confirmationWindow). Maybe something like `rebalancingSetReady`. Or something along those lines.